pkgs/development/libraries/vtk/generic.nix

```diff
@@ -71,6 +74,7 @@ in stdenv.mkDerivation rec {
     "-DCMAKE_INSTALL_INCLUDEDIR=include"
     "-DCMAKE_INSTALL_BINDIR=bin"
   ]
+    ++ optionals isAarch [ "-DVTK_USE_SYSTEM_PNG=ON" ]
```

Comment:
Do we really just want to do this unconditionally like TIFF is being done?

Reply:
I think that would be better because Arch is not that well tested in PRs and it would avoid bit rott.

Reply:
@SuperSandro2000 Great I'll make that change then. Thanks.

Reply:
@SuperSandro2000 Done!